packages/wallet/scripts/validate-env-wallet.js

```diff
@@ -0,0 +1,75 @@
+const path = require("path");
+const dotenv = require("dotenv");
+const fs = require("fs");
+const difference = require("lodash/difference");
+
+const ENV_PATH = path.join(__dirname, "../.env");
+const ENV_TEMPLATE_PATH = path.join(__dirname, "../.env.template");
+const platform = process.argv[2]; // ios or android
+
+function envDiff(arr1, arr2) {
+  const items1 = difference(arr2, arr1);
+  const items2 = difference(arr1, arr2);
+
+  return [items1, items2];
+}
+
+function readEnv(path) {
+  const rawEnvs = fs.readFileSync(path, "utf8");
+
+  return dotenv.parse(rawEnvs);
+}
+
+function validateEnvWallet() {
+  let envs = Object.keys(readEnv(ENV_PATH));
+  let templateEnvs = Object.keys(readEnv(ENV_TEMPLATE_PATH));
+
+  if (!platform) {
+    console.error("Please specify platform value like android or ios");
+    process.exit(1);
+  }
+
+  // Remove some variables from template envs depending on platform
+  templateEnvs = templateEnvs.filter(
+    e => !e.startsWith(platform === "ios" ? "NF_ANDROID" : "NF_IOS"),
+  );
+  envs = envs.filter(e => !e.startsWith(platform === "ios" ? "NF_ANDROID" : "NF_IOS"));
+
+  const invalidNames = envs.filter(env => !env.startsWith("NF_"));
+  if (invalidNames.length > 0) {
+    console.error(
+      `Some of env variables have invalid names (${invalidNames.join(
+        ", ",
+      )}). Name should always start with "NF_".`,
+    );
+    process.exit(1);
+  }
+
+  const [missingEnvs, unknownEnvs] = envDiff(envs, templateEnvs);
+
+  if (missingEnvs.length > 0) {
+    console.error(
+      `Some of env variables are missing (${missingEnvs.join(
+        ", ",
+      )}). Either remove them from ${path.basename(
+        ENV_TEMPLATE_PATH,
+      )} or provide a value in ${path.basename(ENV_PATH)} file.`,
+    );
+    process.exit(1);
+  }
+
+  if (unknownEnvs.length > 0) {
+    console.error(
+      `Some of env variables are unknown (${unknownEnvs.join(
+        ", ",
+      )}). Either remove them from the ${path.basename(
+        ENV_PATH,
+      )} file or provide a default value with description in ${path.basename(ENV_TEMPLATE_PATH)}.`,
+    );
+    process.exit(1);
+  }
+
+  console.log("Env Validation Success!");
+}
+
+validateEnvWallet();
```

Comment:
```suggestion
const platform = process.argv[2]
validateEnvWallet(platform);
```

because this way it is clear that `platform` is coming from outside